jetson/percep/pcl.cpp

```diff
@@ -458,14 +458,15 @@ shared_ptr<pcl::visualization::PCLVisualizer> PCL::createRGBVisualizer() {
 //This function is called in main.cpp
 obstacle_return PCL::pcl_obstacle_detection(shared_ptr<pcl::visualization::PCLVisualizer> viewer) {
     obstacle_return result;
-    PassThroughFilter();
+    PassThroughFilter("z", 7000.0);
+    PassThroughFilter("y", 3000.0);
```

Comment:
How were these determined?

Reply:
We've established that we can trust the ZED depth for up to 7 m which is where the 7k comes from. 3000 was just a placeholder. When we do more outdoor testing will tune that parameter

Reply:
1. You should comment this.
2. This should be in a more convenient location such as a config file.

Reply:
We have a config file in the pipeline

Reply:
I don't know what you mean by "pipeline" but maybe this means my comment should be "this should be in a more convenient locations such as your config file".

Reply:
I mean we're working on it. It's in the works

Reply:
Ahh got it. Then let's do this:

1. Go with the comment for now
2. Include a link to that issue in this PR (I.e. followup on this thread)
3. In that issue, add this item to the description so it is not forgotten.

Reply:
Link to issue: https://github.com/umrover/mrover-workspace/issues/474

Reply:
Don't forget a comment!

Reply:
I thot I did?

Reply:
You're right, my bad on missing that. Can you just throw in the link to the issue for determining the final value for the x value. I think it is best to just have a "finalize all config file values" issue that we do right before competition. Until then just have local changes since this value will change a bunch throughout the year.

Reply:
#517